### PR TITLE
Consolidate more CSS rules, fix inline code on arc-green

### PR DESCRIPTION
**Pull request by silverwind**
- Consolidate various CSS rules into base rules
- Fix inline code in Markdown not having enough contrast on arc-green

Adds one new color variable, `--color-label-active-bg` for the background of active labels.

image
image

### CODE REVIEW

**Review by KN4CK3R — APPROVED** [review]

File: web_src/less/_base.less [hunk 1]
```diff
@@ -393,6 +394,11 @@ a.commit-statuses-trigger {
   background: var(--color-grey);
 }
 
+.ui.active.label {
+  background: var(--color-label-active);
```
Comment (delvh) [inline web_src/less/_base.less]: Should we set the border color as well as it was done previously (even though it was set to the same color)?

File: web_src/less/_base.less [hunk 2]
```diff
@@ -158,6 +158,7 @@
   --color-tooltip-bg: #000000f0;
   --color-tooltip-text: #ffffff;
   --color-header-bar: #ffffff;
+  --color-label-active: #d0d0d0;
```
Comment (delvh) [inline web_src/less/_base.less]: As it is only used for the background, should we perhaps call it `color-background-label-active` or `color-background-active-label`?

**Review by delvh — COMMENTED** [review]
Also, I think it would be a good idea to mark PRs adding new CSS variables as breaking so that theme maintainers can more easily react to changes.

File: web_src/less/_base.less (hunk 2 above)
Comment (silverwind) [reply web_src/less/_base.less]: Hmm yes, we have not been really consistent with labeling background colors in the past, but I guess we could start doing that now.

File: web_src/less/_base.less (hunk 1 above)
Comment (silverwind) [reply web_src/less/_base.less]: IIRC, these do not have a border.

Comment (delvh) [reply web_src/less/_base.less]: https://github.com/go-gitea/gitea/blob/3d92ad8eb9cc255f8153d40e5cbc6a0e9f8fc9eb/web_src/less/themes/theme-arc-green.less#L277

Comment (silverwind) [reply web_src/less/_base.less]: Ah I missed that rule, need to remove it.

Comment (delvh) [reply web_src/less/_base.less]: You removed it already.

Comment (silverwind) [reply web_src/less/_base.less]: Border added. I think those rules are most likely pointless as `.ui.label` has a border value of `0 solid transparent`, meaning no visible border, but I guess to be safe, we override those rules as well.

File: web_src/less/_base.less (hunk 2 above)
Comment (silverwind) [reply web_src/less/_base.less]: Renamed.

**Comment by silverwind** [comment]
> Also, I think it would be a good idea to mark PRs adding new CSS variables as breaking so that theme maintainers can more easily react to changes.

Yes I thought about doing it as well, but decided against it because it's not breaking to users. Our changelogs with breaking stuff are already way too long for users to read, so I thought we should not add any more uninteresting changes like this. Theme authors can easily diff the sources. I always add new vars at the bottom so they can be easily identified.

**Review by wxiaoguang — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**